bad_apps_blog/blog.py

```diff
@@ -52,9 +54,11 @@ def get_post(id, check_author=True):
     ).fetchone()
 
     if post is None:
+        current_app.logger.info(f' [SECURITY] attempted to access post {id} which does not exist')
```

Comment:
It looks like this log message directly puts user input `id` into the log. Couldn't this lead to a log injection attack?

Reply:
Testing this manually shows that this is okay since the Flask view enforces that `id` must be an integer. Should still validate in case a new view is added which does not enforce that `id` is an integer.

Reply:
Most of calls to the logger are directly inside functions that validate id. `get_post()` is only called internally. Adding validation in case new views are implemented without forcing id to be an integer.